lib/chef/azure/commands/enable.rb

```diff
@@ -242,6 +242,7 @@ def escape_runlist(run_list)
         parsedRunlist << "recipe[#{item}]"
       end
     end
+    parsedRunlist

```

Comment:
So we should have some additional commits that catch the bug that occurs when this happens.